src/pubsub/message/sign.js

```diff
@@ -61,6 +64,7 @@ async function verifySignature (message) {
  */
 async function messagePublicKey (message) {
   // should be available in the from property of the message (peer id)
+  // @ts-ignore - from is optional
```

Comment:
its safer to either make it required or protect with and if clause

Reply:
It is optional in fact, but when it reaches this function, it will exist anyway. So, it seems better to do the ts ignore than to protect every usage, which will also lead to a need of new return type